### ISSUE TITLE
Security

### ISSUE THREAD

**Issue by stephenyates-cds** [cds-snc/notification-planning#25]

**Comment by yaelberger-commits** [comment]
@jimleroyer Let's make some time this next week to go through this Epic and refine/update/prioritize. There may be more stories to add to this epic for security as well. 
Also, if we can find some High Priority small sized tasks to bring up to in the Product backlog, those would be nice to have handy during a sprint for when people are waiting on reviews etc.
